src/transaction-creators/nitro-adjudicator.ts

```diff
@@ -0,0 +1,38 @@
+// @ts-ignore
+import NitroAdjudicatorArtifact from '../../build/contracts/NitroAdjudicator.json';
+import {ethers} from 'ethers';
+import {TransactionRequest} from 'ethers/providers';
+import {State, hashState} from '../state';
+import {Outcome, encodeOutcome} from '../outcome';
+import {getChannelId} from '../channel';
+
+// TODO: Currently we are setting some arbitrary gas limit
+// to avoid issues with Ganache sendTransaction and parsing BN.js
+// If we don't set a gas limit some transactions will fail
+const GAS_LIMIT = 3000000;
+
+const NitroAdjudicatorContractInterface = new ethers.utils.Interface(NitroAdjudicatorArtifact.abi);
+
+export function createPushOutcomeTransaction(
+  turnNumRecord: number,
+  finalizesAt: string,
+  state: State,
+  outcome: Outcome,
```

Comment:
As in the other PR, the interface for this should probably ultimately be something like:
```ts
function pushOutcomeTransaction(channelStorage: ChannelStorage)
```